cocos/2d/CCSprite.cpp

```diff
@@ -614,6 +614,17 @@ void Sprite::draw(Renderer *renderer, const Mat4 &transform, uint32_t flags)
     }
 }
 
+TrianglesCommand::Triangles Sprite::getRenderedTriangles() const
+{
+    static unsigned short indices[6] = {0, 1, 2, 3, 2, 1};
```

Comment:
Hi,  I tested the this branch in several android device, including Nexus 5, it works well.
And for you proposed code, I tested it again, and it also works.